frontend/src/screens/wallet/index.tsx

```diff
@@ -16,20 +18,104 @@ import {
   AlertTitle,
 } from "src/components/ui/alert.tsx";
 import { Button } from "src/components/ui/button";
+import { LoadingButton } from "src/components/ui/loading-button";
+import { useToast } from "src/components/ui/use-toast";
+import { ALBY_HIDE_HOSTED_BALANCE_BELOW as ALBY_HIDE_HOSTED_BALANCE_LIMIT } from "src/constants.ts";
+import { useAlbyBalance } from "src/hooks/useAlbyBalance";
 import { useBalances } from "src/hooks/useBalances";
+import { useChannels } from "src/hooks/useChannels";
 import { useInfo } from "src/hooks/useInfo";
+import { request } from "src/utils/request";
 
 function Wallet() {
   const { data: info, hasChannelManagement } = useInfo();
   const { data: balances } = useBalances();
+  const { data: channels } = useChannels();
+  const { data: albyBalance, mutate: reloadAlbyBalance } = useAlbyBalance();
+  const [drainingAlbySharedFunds, setDrainingAlbySharedFunds] =
+    React.useState(false);
+
+  const { toast } = useToast();
 
   if (!info || !balances) {
     return <Loading />;
   }
 
+  const showMigrateCard =
+    albyBalance && albyBalance.sats > ALBY_HIDE_HOSTED_BALANCE_LIMIT;
+
   return (
     <>
       <AppHeader title="Wallet" description="" />
+      {showMigrateCard && (
+        <div className="flex flex-1 items-center justify-center rounded-lg border border-dashed shadow-sm p-8">
+          <div className="flex flex-col items-center gap-1 text-center max-w-md">
+            <SelfCustodyIcon
+              width="211"
+              height="64"
+              viewBox="0 0 211 64"
+              strokeWidth="0"
+              className="text-primary-background"
+            />
+            <h3 className="mt-4 text-lg font-semibold">
+              Your funds ({new Intl.NumberFormat().format(albyBalance?.sats)}{" "}
+              sats) still hosted by Alby.
+            </h3>
+            <p className="text-sm text-muted-foreground">
+              {channels && channels.length > 0
+                ? "Transfer funds to increase your spending capacity."
+                : "Migrate them to your Alby Hub to start using your wallet."}
+            </p>
+            {channels && channels.length > 0 ? (
+              <LoadingButton
+                className="mt-4"
+                loading={drainingAlbySharedFunds}
+                onClick={async () => {
```

Comment:
This is the exact same as "Transfer" button in Node screen. Should I make a `TransferFundsButton`?